src/mixins/viewport/ViewportMixin.js

```diff
@@ -75,11 +68,11 @@ var Viewport = {
     },
 
     subscribeViewportEvents: function () {
-        this.subscribe('scroll', this._detectViewport);
+        this.subscription = subscribe('scroll', this._detectViewport, {throttleRate: 500});
```

Comment:
yes you are right, `scrollEnd` is what I need, updated,

also update README to create a link to `subscribe-ui-event` ;)